prov/efa/src/efa_strerror.c

```diff
@@ -101,6 +102,21 @@ static const char *efa_errno_str(enum efa_errno err)
 		: errno_str[err - FI_EFA_ERRNO_OFFSET];
 }
 
+static const char *efa_debug_str(enum efa_errno err)
+{
+	switch (err) {
+	case FI_EFA_REMOTE_ERROR_BAD_DEST_QPN:
+		return "\nThis error can be caused by a crashed peer. "
+			   "Please verify the receiver status";
+	case FI_EFA_LOCAL_ERROR_UNRESP_REMOTE:
+		return "\nThis error can be caused by incrorrect inbound/outbound "
+			   "rules in the security group. EFA requires \"All traffic\" "
+			   "type allowlisting within the security group";
```

Comment:
Nit: omit preceding newlines from the string literals here. Allow the callers to determine whether or not the extra newline is necessary in their resulting formatted strings.
```suggestion
	case FI_EFA_REMOTE_ERROR_BAD_DEST_QPN:
		return "This error can be caused by a crashed peer. "
			   "Please verify the receiver status";
	case FI_EFA_LOCAL_ERROR_UNRESP_REMOTE:
		return "This error can be caused by incrorrect inbound/outbound "
			   "rules in the security group. EFA requires \"All traffic\" "
			   "type allowlisting within the security group";
```